integration/simulation/network/network.go

```diff
@@ -37,7 +39,7 @@ type RPCHandles struct {
 	//	to mimic user acc interaction via a wallet extension
 	// map of owner addresses to RPC clients for that owner (one per L2 node)
 	// todo: simplify this with a client per node when we have clients that can support multiple wallets
-	VirtualWalletExtensionClients map[string][]rpcclientlib.Client
+	VirtualWalletExtensionClients map[string][]*obsclient.AuthObsClient
```

Comment:
do the `ObscuroClients` still need to be public, after everything will go through the `AuthObsClient`?

`VirtualWalletExtensionClients` should be renamed to `AuthObsClients`?

Reply:
The validation of the network does some general calls to the network where it doesn't need authentication but needs to get a client for each node. I can refactor it to just use the client for each node from one of the wallets but I think separate PR.

Yeah, good shout on the rename, missed that.